docs/the-new-architecture/pillars-fabric-components.md

```diff
@@ -285,8 +285,237 @@ dependencies {
 
 ### Native Code
 
+The last step requires us to write some native code to connect the JS side of our Component to what is offered by the platforms. This process requires two main steps:
+
+- Run the **CodeGen** to see what would be generated
+- Write the native code that will make it work
+
+:::caution
+The code generated by the **CodeGen** in this step should not be committed to the versioning system. React Native apps are able to generate
+the code when the app is built. This allows to avoid any ABI incompatibility and to ensure that a consistent version of the codege is used.
+:::
+
 #### iOS
 
+##### Generate the code
+
+To generate the code starting from the JS specs for iOS, we need to open a terminal and run the following command:
```

Comment:
Yes the command is not specific but android works in a different way, therefore it is iOS-specific. I'm not even sure that it will work for Android at all... 😅